auditbeat/main_test.go

```diff
@@ -41,3 +42,7 @@ func TestSystem(t *testing.T) {
 		main()
 	}
 }
+
+func TestTemplate(t *testing.T) {
```

Comment:
What is the reason you run this here as part of main_test.go?

Reply:
I wanted to have it in a package that contained the same import dependencies as the real application so that all of the same fields.go files would be registered in the asset registry. There's no better place than the main package for this.

Reply:
Got it, makes sense!